modules/virt-creating-configmap.adoc

```diff
@@ -167,103 +267,43 @@ metadata:
-// Windows templates to re-add when Win bug is BZ1856654 is fixed, if we still need custom configmaps
-// "windows_10": "win10"
-// "windows_10x64": "win10"
-// "windows_2003": "win10"
-// "windows_2003x64": "win10"
-// "windows_2008R2x64": "win2k8"
-// "windows_2008": "win2k8"
-// "windows_2008x64": "win2k8"
-// "windows_2012R2x64": "win2k12r2"
-// "windows_2012x64": "win2k12r2"
-// "windows_2016x64": "win2k16"
-// "windows_2019x64": "win2k19"
-// "windows_7": "win10"
-// "windows_7x64": "win10"
-// "windows_8": "win10"
-// "windows_8x64": "win10"
-// "windows_xp": "win10"
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify the REST API name of the RHV VM Windows operating system, for example, `"windows_2016x64"`, and the Windows template.
+endif::[]
 
 . Edit the `kubevirt-hyperconverged-operator.v{HCOVersion}.yaml` file:
 +
 [subs="attributes+"]
 ----
-$ oc edit clusterserviceversion -n openshift-cnv kubevirt-hyperconverged-operator.v2.4.0
+$ oc edit clusterserviceversion -n openshift-cnv kubevirt-hyperconverged-operator.v{HCOVersion}
 ----
 
-. Update the following parameters:
+. Update the following parameters of the `vm-import-operator` deployment:
 +
 [source,yaml]
 ----
-...
-      - name: vm-import-operator
-        spec:
-          replicas: 1
-          selector:
-            matchLabels:
-              name: vm-import-operator
-              operator.v2v.kubevirt.io: ""
-          strategy: {}
-          template:
-            metadata:
-              labels:
-                name: vm-import-operator
-                operator.v2v.kubevirt.io: ""
+            ...
             spec:
               containers:
               - env:
-                - name: DEPLOY_CLUSTER_RESOURCES
-                  value: "true"
-                - name: OPERATOR_VERSION
-                  value: v2.4.0
-                - name: CONTROLLER_IMAGE
-                  value: quay.io/kubevirt/vm-import-controller:v0.0.4
-                - name: PULL_POLICY
-                  value: IfNotPresent
-                - name: WATCH_NAMESPACE
+                ...
                 - name: OS_CONFIGMAP_NAME
                   value: os-configmap <1>
                 - name: OS_CONFIGMAP_NAMESPACE
                   value: default <2>
 ----
-<1> Specify the new ConfigMap name.
-<2> Optional: Specify the new namespace, if relevant.
+<1> Add `value: os-configmap` to the `name: OS_CONFIGMAP_NAME` parameter.
+<2> Optional: You can add this value if you are changed the namespace from `default`.
 
-. Save the `kubevirt-hyperconverged-operator.v2.4.0.yaml` file.
+. Save the `kubevirt-hyperconverged-operator.v{HCOVersion}.yaml` file.
 +
 Updating the `vm-import-operator` deployment updates the `vm-import-controller` ConfigMap.
+
+. Verify that the template appears in the {VirtProductName} web console:
+
+.. Click *Workloads* -> *Virtualization* from the side menu.
+.. Click the *Virtual Machine Templates* tab.
```

Comment:
I think the second part of this step should end with something like "...and find the template in the list."